index.html

```diff
@@ -232,5 +232,42 @@ <h3>Skills</h3>
             </div>
         </div>
     </section>
+    <br>
+    <section>
+        <!-- Contact form -->
+        <div class="div4">
+            <h2 class="color1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        </div>
+        <br>
+        <div class="div5">
+            <form id="contact-form" action="https://formspree.io/f/mlevonav" method="POST">
+                <div>
+                    <input class="input-fields" type="text" name="name" id="name" placeholder="Full Name" max="30" required>
```

Comment:
- [ ] Great job implementing the requested validations! 💯 However, you typed this character length property wrong, it should be `maxlength` instead of `max` for it to work. Please fix this one! 🛠️ 🤓